src/app/shared/layout/header.component.ts

```diff
@@ -135,6 +135,10 @@ export class HeaderComponent implements OnInit {
     this.electronService.openLink('https://github.com/'.concat(this.githubService.getRepoURL()).concat(issueUrl));
   }
 
+  openIssueTracker() {
+    this.electronService.openLink('https://github.com/CATcher-org/CATcher/issues');
```

Comment:
maybe instead of hard coding this, maybe we can store this in a constants file =)

Reply:
Can we have it as a `const` variable at the top of this file?
It seems that constants files are created only for tests currently, so for e.g. `tests\constants\label.constants.ts` and these constants are only used in testing.

Reply:
Updated